package.json

```diff
@@ -198,7 +200,7 @@
 		"react-instantsearch-hooks-web": "^6.33.0",
 		"react-intersection-observer": "^8.33.1",
 		"react-player": "^2.9.0",
-		"rehype-sanitize": "^4.0.0",
+		"rehype-sanitize": "^5.0.1",
```

Comment:
I've upgraded to the latest `rehype-sanitize`, as it allows for regex-based filtering of properties. This lets us allow `data-text-content` properties needed for permalink generation, while ensuring those properties contain only alphanumeric characters.

I've also added `hast-util-sanitize` as a dependency, as we do import the `defaultSchema` we use for sanitization from that package directly.